README.md

```diff
@@ -78,6 +80,15 @@ instead of `bin`.
   
 * For help use `pyspdxtools --help`
 
+3. **VISUALIZATION** (optional feature)
+
+* Make sure you install the optional dependencies `networkx` and `pygraphviz`. To do so run `pip install ".[visualization]"`.
+* Use `pyspdxtools -i <input_file> --visualize <output_file>` where `<output_file>` is a valid output for `pygraphviz` (check 
+  the documentation [here](https://pygraphviz.github.io/documentation/stable/reference/agraph.html#pygraphviz.AGraph.draw)). 
+* If you are using a source distribution, try running
+  `pyspdxtools -i tests/data/formats/SPDXJSONExample-v2.3.spdx.json --visualize SPDXJSONExample-v2.3.spdx.png` to generate 
+  a png with an overview of the structure of the example file.  
```

Comment:
We should provide an example of this somewhere so users will know if this feature is something they want before they actually have to install all this.

Reply:
I added an example png in the readme, please have a look if this fits your intention.